WordPressKit/BlogServiceRemoteREST.m

```diff
@@ -287,6 +287,25 @@ - (void)fetchSiteInfoForAddress:(NSString *)siteAddress
                           }];
 }
 
+- (void)fetchUnauthenticatedSiteInfoForAddress:(NSString *)siteAddress
+                        success:(void(^)(NSDictionary *siteInfoDict))success
+                        failure:(void (^)(NSError *error))failure
+{
+    NSString *path = [self pathForEndpoint:@"connect/site-info" withVersion:ServiceRemoteWordPressComRESTApiVersion_1_1];
+    NSURL *siteURL = [NSURL URLWithString:siteAddress];
+
+    [self.wordPressComRestApi GET:path
+                       parameters:@{ @"url": siteURL.absoluteString }
+                          success:^(id responseObject, NSHTTPURLResponse *httpResponse) {
+                              if (success) {
+                                  success((NSDictionary *)responseObject);
+                                  return;
```

Comment:
Nitpicky of me, but we probably don't need the `return;` here.

Reply:
Do we need it in fetchSiteInfo?

Reply:
Doesn't look like we need it there either, but no worries!